src/main/java/com/flipkart/ranger/healthcheck/Healthcheck.java

```diff
@@ -16,6 +16,15 @@
 
 package com.flipkart.ranger.healthcheck;
 
+/**
+ * An interface used to check the health of the service
+ */
 public interface Healthcheck {
+
+    /**
+     * return healthy/unhealthy status of the service
+     *
+     * @return health status
+     */
     public HealthcheckStatus check();
```

Comment:
Yup, but ranger currently supports java 7 and above. Will make sure to change this, when we completely move away from java 7.